README.md

```diff
@@ -23,6 +24,16 @@ You can run the sandbox on your system, or in a virtualbox VM using Vagrant. For
   * phpunit 3.6+ (optional)
   * composer
 
+## Setup with vagrant
+
+You simply need to run **vagrant up** in your project directory and the setup configures everything.
```

Comment:
hmm, it should be told a bit more about the doctrine-dbal and the jackrabbit implementations here...